evmcore/tx_pool_test.go

```diff
@@ -1541,129 +1548,130 @@ func TestTransactionPendingMinimumAllowance(t *testing.T) {
 	}
 }
 
+// TODO: this test is rewritten in replacements test
```

Comment:
Left-over TODO / missing code cleanup?

Also: where is this replacement test?

Reply:
here : https://github.com/Fantom-foundation/Sonic/pull/329